### ISSUE TITLE
Search param `split_join_tokens` is not supported

### ISSUE THREAD

**Issue by cmigazzi** [typesense/typesense-go#111]
## Description

The query param `split_join_tokens` is not supported by `SearchCollectionParams`. I've checked the [struct](https://github.com/typesense/typesense-go/blob/275ab77474322015480e51498fca7a419dfff056/typesense/api/types_gen.go#L469-L505) and haven't  found the param definition.

## Steps to reproduce
Try to create search collection params with `SplitJoinTokens`:

```go
searchParameters := &api.SearchCollectionParams{
	Q:               "test",
	QueryBy:         "keywords",
	SplitJoinTokens: "always",
}
```

## Expected Behavior

No error and being able to use `split_join_tokens` as specified in Typesense [documentation](https://typesense.org/docs/0.23.1/api/search.html#query-parameters)

## Actual Behavior

Trigger an error: `unknown field 'SplitJoinTokens' in struct literal of type api.SearchCollectionParams`

## Metadata

**Typsense Version**: 0.23.0

**Comment by cmigazzi** [comment]
I've just opened a PR to add these parameters: https://github.com/typesense/typesense-api-spec/pull/29

**Comment by kishorenc** [comment]
Thanks, I've merged that PR and have generated updated client code and published it as `v0.6.0`.
